### ISSUE TITLE
Please change the name of the project to lispcookbook!

### ISSUE THREAD

**Issue by daninus14** [LispCookbook/cl-cookbook#562]
Hi,

Can you please change the name of the repository to be lispcookbook.github.io??

That way the address will be https://lispcookbook.github.io/ instead of https://lispcookbook.github.io/

It's a bit annoying every time my browser navigates to the https://lispcookbook.github.io/ instead of https://lispcookbook.github.io/cl-cookbook/, specially when there are no links there to go to the correct page...

Thanks!

**Comment by vindarel** [comment]
I just talked about this with a friend yesterday; it may be too late to rename the project but we can create another Github pages repo with a meta redirect in its index.

**Comment by daninus14** [comment]
I think the only issues with renaming a repository are related to github actions, everything else including issues, wikis, followers, stars, and probably forks will stay. See https://docs.github.com/en/repositories/creating-and-managing-repositories/renaming-a-repository

From what I can see the actions you have are funding, which doesn't seem to be dependent on a name (unless I got it wrong) and the build pages, which is the whole point of the rename.

I thought many times before to submit this issue but refrained from doing so, but it really only takes a sec, that's why I asked.

**Comment by vindarel** [comment]
What about the deployed pages?

    https://lispcookbook.github.io/cl-cookbook/getting-started.html

would become

    https://lispcookbook.github.io/getting-started.html

right?

**Comment by daninus14** [comment]
yeah, I think so. Ah are you worried about all the links around the web pointing to the previous URL?

**Comment by daninus14** [comment]
I wonder if in the github actions build workflow, whenever you push to the project, you can make it push as well to the other repository as well. That way one repo will be a mirror of the other, which will get pushed to every time there is a push in the first repository. Each repo will have the regular github action build github pages setup, so everything would work well.

That way, since it doesn't look like redirecting is possible, at least the content will be all the same.
